aiohttp/connector.py

```diff
@@ -931,13 +934,7 @@ def _get_ssl_context(self, req: ClientRequest) -> Optional[SSLContext]:
             sslcontext = req.ssl
             if isinstance(sslcontext, ssl.SSLContext):
                 return sslcontext
-            if sslcontext is not None:
-                # not verified or fingerprinted
-                return self._make_ssl_context(False)
-            sslcontext = self._ssl
-            if isinstance(sslcontext, ssl.SSLContext):
-                return sslcontext
-            if sslcontext is not None:
```

Comment:
None of this looks like it should be removed, only the `is None` condition changed.

Reply:
They are duplicated code. (exactly same as code above)

Reply:
No it's not, it checks req.ssl and then self._ssl.